ipanonymizer/anonymizer.go

```diff
@@ -0,0 +1,97 @@
+// Package ipanonymizer provides functionality to anonymize IP-Addresses.
+package ipanonymizer
+
+import (
+	"errors"
+	"net"
+)
+
+var (
+	defaultIPv4Mask = net.IPv4Mask(255, 255, 255, 0)                                                     // /24
+	defaultIPv6Mask = net.IPMask{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0, 0, 0, 0, 0, 0, 0, 0} // /64
```

Comment:
```suggestion
	defaultIPv4Mask = net.CIDRMask(24, 32)
	defaultIPv6Mask = net.CIDRMask(64, 128)
```

Reply:
CIDRMask is easier to comprehend (if you already know what the args mean).

The current way is the same then how package vars are initialized in the `net` package and is more efficient.
These are also private variables and a user of the package never has to read them.
I keep the current variant. :-)